### ISSUE TITLE
VSCode ESLint extension throws TypeError on indent

### ISSUE THREAD

**Issue by benwebbbenwebb** [eslint-stylistic/eslint-stylistic#47]
### Problem
Using eslint-stylistic (v1.0.0) is throwing an error on my migrated indent rule while using VScode's ESLint extension (v2.4.2).

Note: running eslint directly in terminal (eslint .) has no such error, and correctly shows linting issues.

Removing the rule entirely then generates a similar looking (but not the same) failure on a different rule... my best guess is that 'sourceCode' isn't being passed correctly? Is this something you can help look at or should I discuss with the ESLint Extension owner?

Thanks!

### Rule
```json
    "@stylistic/indent": [
        "error",
        "tab",
        { "SwitchCase": 1 } 
    ],
```

### Error
```
[Error - 22:56:18] An unexpected error occurred:
[Error - 22:56:18] TypeError: Error while loading rule '@stylistic/indent': Cannot read properties of undefined (reading 'tokensAndComments')
Occurred while linting /Users/malcolm/dev/projects/homebridge-sensibo-ac-nitaybz/homekit/AirConditioner.js
    at new TokenInfo (/Users/malcolm/dev/projects/homebridge-sensibo-ac-nitaybz/node_modules/@stylistic/eslint-plugin-js/dist/indent.js:189:31)
    at Object.create (/Users/malcolm/dev/projects/homebridge-sensibo-ac-nitaybz/node_modules/@stylistic/eslint-plugin-js/dist/indent.js:688:23)
    at create (/Users/malcolm/dev/projects/homebridge-sensibo-ac-nitaybz/node_modules/@stylistic/eslint-plugin-ts/dist/indent.js:108:28)
    at Object.create (/Users/malcolm/dev/projects/homebridge-sensibo-ac-nitaybz/node_modules/@typescript-eslint/utils/dist/eslint-utils/RuleCreator.js:38:20)
    at createRuleListeners (/Users/malcolm/dev/projects/homebridge-sensibo-ac-nitaybz/node_modules/eslint/lib/linter/linter.js:926:21)
    at /Users/malcolm/dev/projects/homebridge-sensibo-ac-nitaybz/node_modules/eslint/lib/linter/linter.js:1108:110
    at Array.forEach (<anonymous>)
    at runRules (/Users/malcolm/dev/projects/homebridge-sensibo-ac-nitaybz/node_modules/eslint/lib/linter/linter.js:1045:34)
    at Linter._verifyWithoutProcessors (/Users/malcolm/dev/projects/homebridge-sensibo-ac-nitaybz/node_modules/eslint/lib/linter/linter.js:1397:31)
    at Linter._verifyWithConfigArray (/Users/malcolm/dev/projects/homebridge-sensibo-ac-nitaybz/node_modules/eslint/lib/linter/linter.js:1772:21)
```
